wpilibNewCommands/src/main/java/edu/wpi/first/wpilibj2/command/CommandScheduler.java

```diff
@@ -430,6 +431,32 @@ public Command getDefaultCommand(Subsystem subsystem) {
     return m_subsystems.get(subsystem);
   }
 
+  private void cancel(Command command) {
```

Comment:
Having a ridiculous amount of overloads named `cancel` isn't good, especially when there's both single-arg and vararg  overloads. Rename this?